vendor.conf

```diff
@@ -37,7 +37,7 @@ github.com/mailru/easyjson d5b7844b561a7bc640052f1b935f7b800330d7e0
 github.com/Microsoft/go-winio v0.4.4
 github.com/opencontainers/go-digest 21dfd564fd89c944783d00d069f33e3e7123c448
 github.com/opencontainers/image-spec v1.0.0
-github.com/opencontainers/runc 593914b8bd5448a93f7c3e4902a03408b6d5c0ce
+github.com/opencontainers/runc 0351df1c5a66838d0c392b4ac4cf9450de844e2d
```

Comment:
See: https://github.com/containerd/containerd/commit/e789135aa3ffa335b6a49ec52733dc13783747a2

Reply:
Will update.

Reply:
Done.

Reply:
We can't update now. The `docker/pkg/signal` package we are using is still using `unix.SIGUNUSED`.

However, the new `x/sys` package removed that https://github.com/containerd/containerd/commit/727fd599fba8218274a4acc3c1014823875e0e05.

Reply:
I'll file an issue.

Reply:
yeah saw some serious changes.. thus the reason for pointing it out. To bad about the docker dependency.

Reply:
Filed #333 to track this.